### PR TITLE
Update Kujira gas tokens

### PR DESCRIPTION
**Pull request by codehans**

### CODE REVIEW

File: kujira/chain.json [hunk 1]
```diff
@@ -9,18 +9,122 @@
...
+      },
+      {
+        "denom": "factory/kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7/uusk",
+        "fixed_min_gas_price": 0.01186,
+        "low_gas_price": 0.01186,
+        "average_gas_price": 0.01779,
+        "high_gas_price": 0.02372
+      },
+      {
+        "denom": "ibc/295548A78785A1007F232DE286149A6FF512F180AF5657780FC89C009E2C348F",
```
Comment (JeremyParish69) [inline kujira/chain.json:31]: These denoms are not found. Please ensure all these assets are defined in kujira's assetlist.json

**Comment by codehans** [comment]
@JeremyParish69 all updated

File: kujira/assetlist.json [hunk 2]
```diff
@@ -385,6 +385,572 @@
...
+      "name": "Wormhole Solana",
+      "display": "sol",
+      "symbol": "whSOL",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "wormchain",
+            "base_denom": "factory/wormhole14ejqjyq8um4p3xfqj74yld5waqljf88fz25yxnma0cngspxe3les00fpjx/8sYgCzLRJC3J7qPn2bNbx6PiGcarhyx8rBhVaNnfvHCA",
+            "channel_id": "channel-9"
```
Comment (JeremyParish69) [inline kujira/assetlist.json:410]: We are also going to need the IBC connections registered. for this one, we need /_IBC/kujira-solana.json

File: kujira/assetlist.json [hunk 3]
```diff
@@ -385,6 +385,572 @@
...
+      "name": "Comdex",
+      "display": "cmdx",
+      "symbol": "CMDX",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "comdex",
+            "base_denom": "ucmdx",
+            "channel_id": "channel-31"
```
Comment (JeremyParish69) [inline kujira/assetlist.json:671]: will also need comdex-kujira.json

File: kujira/assetlist.json [hunk 4]
```diff
@@ -385,6 +385,572 @@
...
+      "name": "Juno",
+      "display": "juno",
+      "symbol": "JUNO",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "juno",
+            "base_denom": "ujuno",
+            "channel_id": "channel-87"
```
Comment (JeremyParish69) [inline kujira/assetlist.json:715]: and juno-kujira.json

**Comment by codehans** [comment]
@JeremyParish69 added

**Comment by JeremyParish69** [comment]
Had to change 'wormchain' to 'gateway'

**Pull request merged**